### PR TITLE
Fix unit test error of uefi-pi

### LINKED ISSUE #153: [Bug] Unit test error of td-uefi-pi/src/fv.rs
td-uefi-pi/src/fv.rs

```
failures:

failures:
    fv::test::test_get_file_from_fv
    fv::test::test_get_image_from_fv
```

### PR DESCRIPTION
**Pull request by liuw1**
Fix #153 

Fix test seeds fuzzing/in/fuzz_fv_parser/fv_buffer and
fuzzing/in/fuzz_cfv_parser/cfv. Old files has uncorrect
checksum value in FV header.

Signed-off-by: Wei Liu <wei3.liu@intel.com>

### CODE REVIEW

**Comment by jyao1** [comment]
I don't understand the comment - Old files has uncorrect checksum value in FV header.

The patch just fixed the file name...

**Comment by liuw1** [comment]
Sorry, my patch missed the files.

**Comment by jiangliu** [comment]
It seems like a rename, and I feel "cfv.bin" is better than "cfv" too.

File: td-uefi-pi/src/fv.rs [hunk 1]
```diff
@@ -271,7 +271,7 @@ mod test {
 
     #[test]
     fn test_get_file_from_fv_with_wrong_guid() {
-        let bytes = include_bytes!("../../fuzzing/in/fuzz_cfv_parser/cfv.bin");
+        let bytes = include_bytes!("../../fuzzing/in/fuzz_cfv_parser/cfv");
```
[the remark this answers is not in the document]
Comment (liuw1) [reply td-uefi-pi/src/fv.rs:274]: Yes, I think we can. Maybe we can do this change in another patch.

**Comment by jyao1** [comment]
please also add description on how to generate this binary file.

**Comment by liuw1** [comment]
> It seems like a rename, and I feel "cfv.bin" is better than "cfv" too.

I regenerated cfv binary to fix the checksum value in FVH. Since the .gitignore file contains '*.bin', so I also renamed the file.

**Comment by liuw1** [comment]
> please also add description on how to generate this binary file.
I have added description in commit message of https://github.com/confidential-containers/td-shim/commit/7ce6944a9b0308904460540b533b55124f9274d5 before. Maybe we can add one detail doc to describe how to generate each fuzzing test binary later.

**Review by jiangliu — APPROVED** [review]

**Pull request merged**